src/Hangman/Domain/Events/NewGameStarted.event.ts

```diff
@@ -1,16 +1,26 @@
 import { StorableEvent } from '@peterdijk/nestjs-eventstoredb';
+import { MaxGuesses } from '../ValueObjects/MaxGuesses.value-object';
+import { Word } from '../ValueObjects/Word.value-object';
 
 export class NewGameStartedEvent extends StorableEvent {
-  public readonly eventVersion = 1;
+  public eventVersion = 1;
+  /* Changelog
+   * v2: wordToGuess and maxGuesses are value objects
+
+  */
 
   constructor(
     public readonly id: string,
     public readonly playerId: string,
-    public readonly wordToGuess: string,
+    public readonly wordToGuess: string, // should the type in the event be that of the ValueObject or the string value ?
```

Comment:
when trying this ran into issue that in event it's stored as for example wordToGuess.props.value
we should only access the VA through it's getter, so I think better just store the string value in the event

what do you think @weemen